wolfssl/openssl/evp.h

```diff
@@ -280,6 +297,7 @@ WOLFSSL_API int  wolfSSL_EVP_Cipher(WOLFSSL_EVP_CIPHER_CTX* ctx,
                           unsigned char* dst, unsigned char* src,
                           unsigned int len);
 
+WOLFSSL_API const WOLFSSL_EVP_CIPHER* wolfSSL_EVP_get_cipherbynid(int);
```

Comment:
Please add the new function in to automated tests. Follow the pattern in <wolfssl-root>/tests/api.c and add in a new compatibility layer test function or add it to an existing test function in api.c if it fits in with one.